datadog/resource_datadog_monitor.go

```diff
@@ -344,6 +360,12 @@ func resourceDatadogMonitorUpdate(d *schema.ResourceData, meta interface{}) erro
 		if thresholds["critical"] != nil {
 			o.Thresholds.SetCritical(json.Number(thresholds["critical"].(string)))
 		}
+		if thresholds["warning"] != nil {
```

Comment:
I might be wrong, but shouldn't this be `warning_recovery`?

Reply:
fixed